src/Dapr.Client/DaprClient.cs

```diff
@@ -431,5 +431,12 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             string storeName,
             IReadOnlyDictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Sets the Dapr Api Token which will be added to the header in all requests
+        /// </summary>
+        /// <param name="apiToken">Dapr Api Token value.</param>
+        public abstract void SetDaprApiToken(
```

Comment:
This should be part of the builder, not part of the client. It's something that's fixed for the lifetime of the app.